dashboard/app/reporting_test.go

```diff
@@ -460,3 +460,24 @@ func TestReportingFilter(t *testing.T) {
 	rep4 := c.client.pollBug()
 	c.expectEQ(string(rep4.Config), `{"Index":2}`)
 }
+
+func TestMachineInfo(t *testing.T) {
+	c := NewCtx(t)
```

Comment:
How should I incorporate the new machineInfo field into the existing tests? Change all the test so that they include the new field?

Reply:
Changing all tests is unnecessary (too time consuming and not useful). I think there is 1 test that checks all fields of BugReport, that test needs to be changed. + some additional tests that test specific aspects of machine info handling, if necessary.